### PR TITLE
MGMT-841 Assisted installer should use kubeconfig-loopback

### PR DESCRIPTION
**Pull request by eranco74**

### CODE REVIEW

File: src/main/main.go [hunk 1]
```diff
@@ -27,44 +25,11 @@ func main() {
...
-func getKubeconfig(logger *logrus.Logger) (string, error) {
-	o := ops.NewOps(logger)
-	ic := inventory_client.CreateBmInventoryClient()
-	if err := o.Mkdir(installer.InstallDir); err != nil {
-		logger.Errorf("Failed to create install dir: %s", err)
-		return "", err
-	}
-	filename := "kubeconfig"
-	dest := filepath.Join(installer.InstallDir, filename)
-	err := ic.DownloadFile(filename, dest)
```
Comment (ronniel1) [inline src/main/main.go:64]: Is there no need to download the kubeconfig anymore?

Comment (eranco74) [reply src/main/main.go:64]: No, we use the one we get from the ignition (kubeconfig-loopback)

Comment (ronniel1) [reply src/main/main.go:64]: So who uses the kubeconfig that is in the cloud?

Comment (eranco74) [reply src/main/main.go:64]: The user

**Pull request merged**